src/Rules/Operators/InvalidUnaryOperationRule.php

```diff
@@ -23,15 +23,24 @@ public function processNode(\PhpParser\Node $node, Scope $scope): array
 		if (
 			!$node instanceof \PhpParser\Node\Expr\UnaryPlus
 			&& !$node instanceof \PhpParser\Node\Expr\UnaryMinus
+			&& !$node instanceof \PhpParser\Node\Expr\BitwiseNot
 		) {
 			return [];
 		}
 
 		if ($scope->getType($node) instanceof ErrorType) {
+
+			if ($node instanceof \PhpParser\Node\Expr\UnaryPlus) {
+				$operator = '+';
+			} elseif ($node instanceof \PhpParser\Node\Expr\UnaryMinus) {
+				$operator = '-';
+			} else {
+				$operator = '~';
```

Comment:
The changed rule should be in separate commit along with the related test changes.